src/jvmMain/kotlin/UseIngredient.kt

```diff
@@ -0,0 +1,2 @@
+val lettuce = Ingredient("lettuce")
+val arugula = Ingredient.name.modify(lettuce) { "arugula" }
```

Comment:
Without `kotlin.srcDir` above, it won't recognize the companion `name` and the compiler will complain.